services/embedService.js

```diff
@@ -6,9 +6,9 @@ const argsTitle = (args) => {
     : argsArray;
 };
 
-const embedTemplate = (msg) => {
+const embedTemplate = (msg, color = 16750462) => {
   return {
-    color: 16750462,
+    color,
```

Comment:
I don't think this is necessary, later when `Object.assign(embedTemplate(msg), options)` is done, the duplicate keys from the 2nd arg (options) will override the first I believe.

Reply:
True.